code/modules/awaymissions/corpse.dm

```diff
@@ -295,3 +381,48 @@
 	mob_species = "abductor"
 	corpseuniform = /obj/item/clothing/under/color/grey
 	corpseshoes = /obj/item/clothing/shoes/combat
+
+
+
+////////////////Random Corpse spawner//////////////////
+//Allows you to place down a random spawner object and have the map randomly choose one from a list, very much like maint loot spawners.
+
+
+/obj/effect/landmark/corpse/random
+	icon = 'icons/mob/screen_gen.dmi'
+	icon_state = "x2"
+	color = "#00FF00"
+	var/list/loot			//a list of possible items to spawn e.g. list(/obj/item, /obj/structure, /obj/effect)

```

Comment:
Tigercat meant it should probably be 
`var/list/loot = new list()`

But I'm pretty sure that won't work, as you can only use static values here.